app/configurator/components/ui-helpers.ts

```diff
@@ -145,7 +145,7 @@ export const getIconName = (name: string): IconName => {
     case "y":
       return "yAxis";
     case "segment":
```

Comment:
I am curious, why this change ? It seems like Color is more user friendly.

Reply:
We decided to change it, as this section grew significantly over the time - now we also have the `Chart mode` here, as color technically is a sub-option of segmentation (color palette). Not sure if this explanation makes sense, but segmentation and color and not on the same level conceptually, at least we came to such conclusion 😅